l2geth/rollup/rcfg/system_address.go

```diff
@@ -0,0 +1,87 @@
+package rcfg
+
+import (
+	"math/big"
+	"os"
+
+	"github.com/ethereum-optimism/optimism/l2geth/common"
+)
+
+// SystemAddress0 is the first deployable system address.
+var SystemAddress0 = common.HexToAddress("0x4200000000000000000000000000000000000042")
+
+// SystemAddress1 is the second deployable system address.
+var SystemAddress1 = common.HexToAddress("0x4200000000000000000000000000000000000014")
+
+// ZeroSystemAddress is the emprt system address.
+var ZeroSystemAddress common.Address
```

Comment:
Mostly just a naive go question for myself, does this just return `common.HexToAddress("0x0000000000000000000000000000000000000000")`?

Reply:
Yep! I'm using this so that I don't have to use `common.Address{}` everywhere.

Reply:
Yes it does, that is the zero value of `common.Address`